### ISSUE TITLE
1-dimensional persistence diagram of image data

### ISSUE THREAD

**Issue by AshleyChraya** [scikit-tda/scikit-tda#22]
Hi,

Is there a way to create 1-dimensional persistence diagram using sublevelset filtration or lower star filtration on image data? I think it is not possible to construct 1D PD with lower_star_img. Currently, I am able to construct 0D PD using lower_star_img, but I can't find a way to make 1D PDs. Any help would be greatly appreciated.

Thank you.

**Comment by ctralie** [comment]
That's right!  This feature is not currently supported.  Pull requests are
welcome, or you may want to check out the cubical complex functionality in
GUDHI
https://gudhi.inria.fr/python/latest/cubical_complex_user.html

**Comment by ctralie** [comment]
Actually, though, I should mention that there is a mathematical "hack" you
can do if you're just focused on 1D persistence on images.  If you filter
from top to bottom but do H0, this is equivalent to H1 by Alexander
Duality.  You can get the current code to do this by simply passing the
negative of your image to it.  Hope that helps!
